pkg/project/auth/cache_test.go

```diff
@@ -0,0 +1,143 @@
+package auth
+
+import (
```

Comment:
Yes. Look at how upstream fake has a number of those static objects.  It lets you mock client behavior.  

Sent from my iPhone

> On Feb 20, 2015, at 8:07 AM, David Eads <notifications@github.com> wrote:
> 
> In pkg/project/auth/cache_test.go:
> 
> > @@ -0,0 +1,143 @@
> > +package auth
> > +
> > +import (
> I'm not seeing it. I just see a static list being copied back: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/client/fake_namespaces.go#L33, that's better than the empty ones in origin?
> 
> —
> Reply to this email directly or view it on GitHub.
>